site/deploy/about/about.md

```diff
@@ -34,6 +34,23 @@ The Berkeley Center for Law and Technology supported this research in part in co
 
 This work was supported by the European Union’s Seventh Framework Programme (FP7), via the [PRIPARE project](http://pripareproject.eu/).
 
+## Attribution
```

Comment:
I think these changes need to be made in `site/content/about/about.md` rather than in this file. (I believe this file is just copied over in the build process.)

Related: we'll want to make sure this merges with other recent changes to the About file. I'm not sure if we need to re-base patch:3 to include updates from master or if the merge will work smoothly anyway.